packages/legacy/core/App/container-api.ts

```diff
@@ -7,7 +7,7 @@ import {
 } from '@credo-ts/core'
 import { IndyVdrPoolConfig } from '@credo-ts/indy-vdr'
 import { ProofRequestTemplate } from '@hyperledger/aries-bifold-verifier'
-import { OCABundleResolverType } from '@hyperledger/aries-oca/build/legacy'
+import { RemoteOCABundleResolver } from '@hyperledger/aries-oca/build/legacy'
```

Comment:
Not everyone will be using Remote OCA so is there a more generic type that will work? Why does `OCABundleResolverType` not work here?

Reply:
`OCABundleResolverType` does not have the `checkForUpdates` method used in the splash screen. It's okay though I'll use `OCABundleResolverType` in the container and then cast it to RemoteOCABundleResolverType in the splash screen